htdocs/expedition/class/expedition.class.php

```diff
@@ -1724,12 +1724,12 @@ public function fetch_lines()
 
 			$shipmentlinebatch = new ExpeditionLineBatch($this->db);
 
+			$line = new ExpeditionLigne($this->db); // always set $line for PHAN analyser. @phan-var-force muse be used after an assignation, and there is no assignation for $line.
```

Comment:
@phan-var-force (l.1732) must be used after an assignation, and we don't have any assignation for $line at the beginning of the loop.

This is the solution I found to pass Phan tests.

source : https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#inline-type-checks-via-string-literals